client/state/stats/lists/reducer.js

```diff
@@ -84,6 +85,27 @@ export const items = withSchemaValidation( itemSchema, ( state = {}, action ) =>
 					},
 				},
 			};
+		}
+
+		case ALL_SITES_STATS_RECEIVE: {
+			const { query, data } = action;
+			const newState = { ...state };
+
+			for ( const siteId in data.stats ) {
+				const siteData = data.stats[ siteId ];
+				const statType = 'statsSummary'; // ??
+				const queryKey = getSerializedStatsQuery( query );
+
+				newState[ siteId ] = {
+					...newState[ siteId ],
+					[ statType ]: {
+						...get( newState, [ siteId, statType ] ),
```

Comment:
Would be perfect if we don't introduce yet another `lodash` usage. I realize it's already been used in this module, but we've aiming to reduce its usage: p4TIVU-9Bf-p2